### ISSUE TITLE
Loading chunks and config editing  1.10 minecraft skyfactory 

### ISSUE THREAD

**Issue by wolfhound871** [LatvianModder/FTBUtilities#347]
loading chunks when making it so players are not aloud to chunk load they can anyways and it still loads them and editing the config is not possible in the file it reverts back to default only possible in game and in game i cant control how many claim blocks or chunk claim blocks players get

**Comment by krum85** [comment]
Having the same problem when it comes to the rank config. It reverts to the default when I start my Server. Currently running Direwolf20 v1.4.1 for minecraft 1.10. Here is a pastebin of how I try to set up my default_rank_config.json file: http://pastebin.com/veUVMd0F

**Comment by LatvianModder** [comment]
You should move that to ranks.json and set it like this: 
```
{
  "default_rank": "player",
  "ranks": {//here goes that whole file you pastebined//}
}
```

**Comment by LatvianModder** [comment]
Fixed in 3.3.0+, you can now change things very easily
